### PR TITLE
[Processing] some tests added

### PR DESCRIPTION
**Pull request by ghtmtt**

### CODE REVIEW

**Comment by volaya** [comment]
nice work!

tests seem to be failing, and it looks like a precision issue. @m-kuhn, any idea?

**Comment by ghtmtt** [comment]
ehi.. yep it should be a geometry precision issue.. I looked at other tests and with the new commit it should not complain.. 
Strange (at least for me) is that running test on my local machine I don't have errors.

I just get an error of the test **qgis:createpointslayerfromtable**:

` F...Warning 1: Geometry type None is not compatible with GEOMETRY=AS_XYZ.`


 so I have to comment this test, run locally, uncomment and commit

File: Testing/Temporary/CTestCostData.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+---
```
Comment (m-kuhn) [inline Testing/Temporary/CTestCostData.txt]: Can you remove this file from the pull request

    git rm Testing/Temporary/CTestCostData.txt

and add the following line to the end of the .gitignore file

    Testing/*

Comment (ghtmtt) [inline Testing/Temporary/CTestCostData.txt]: done!

**Comment by ghtmtt** [comment]
travis fails again.. I looked deeper into the 2 failed tests and it seems that the geometry fails at the 6 decimal digit. I set the parameter from 7 to 5.. Hoping it works now

**Comment by ghtmtt** [comment]
No fails.. Can we merge?

**Comment by m-kuhn** [comment]
Thank you @ghtmtt !

**Pull request merged**